src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -283,9 +283,11 @@ g.test('depth_test_fail')
     t.runDepthStateTest(testStates, _expectedColor);
   });
 
-// Use a depth value that's not exactly 0.5 because it is exactly between two depth16unorm value and
-// can get rounded either way (and a different way between shaders and clearDepthValue).
-const kMiddleDepthValue = 0.5001;
+// Use a depth value of 0.4, which is exactly representable in 16 bits unrom (26214 / (2^16-1)),
+// 24 bits unorm (6710886 / (2^24-1)), and 32 bits unorm (1717986918 / (2^32-1)).
```

Comment:
For depth we don't have any 32-bit unorm formats, only 32-bit _float_ formats. depth24plus is defined as being either 24unorm or 32float:

> The depth component of the "depth24plus" and "depth24plus-stencil8" formats may be implemented as either a 24-bit depth value or a "depth32float" value.

0.4 is not representable in 32float (it's 0.4000000059604644775390625), but probably 32float has enough precision already that it doesn't matter, so my suggestion is really just about this comment.

```suggestion
// Use a depth value of 0.4, which is exactly representable in depth16unorm (26214 / (2^16-1))
// and depth24unorm (6710886 / (2^24-1)), and closely approximated in depth32float
// (0.4000000059604644775390625).
```